edocumentation-service/src/main/java/org/dcsa/edocumentation/service/unofficial/UnofficialShippingInstructionService.java

```diff
@@ -8,10 +8,9 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.dcsa.edocumentation.domain.persistence.entity.ShippingInstruction;
-import org.dcsa.edocumentation.infra.enums.EblDocumentStatus;
 import org.dcsa.edocumentation.domain.persistence.entity.unofficial.ValidationResult;
 import org.dcsa.edocumentation.domain.persistence.repository.ShippingInstructionRepository;
-import org.dcsa.edocumentation.service.mapping.DocumentStatusMapper;
+import org.dcsa.edocumentation.infra.enums.EblDocumentStatus;
```

Comment:
what does the infra in the package name means ?

Reply:
This change was caused by automatic organizing of imports and was not relevant to the change I wanted to make. I have undo this change, so the import is no longer changed in this PR and therefore no longer a delta from master.